.github/workflows/build.yaml

```
@@ -324,6 +324,7 @@ jobs:
       matrix:
         repo:
           - entitlement-pdp
+          - keycloak-ers
```

```suggestion
          - entity-resolution
```

Can we change the name of the folder/project to `entity-resolution` rather than `keycloak-ers`?

---

Fixed